drivers/Aqara/aqara-lock/src/init.lua

```diff
@@ -67,41 +69,60 @@ local function toHex(value, length)
 end
 
 local function event_lock_handler(driver, device, evt_name, evt_value)
-  if evt_value == 1 then
+  if evt_value == 0x1 then
     device:emit_event(Lock.lock(evt_name))
-    device:emit_event(TamperAlert.tamper.clear())
+    device:emit_event(LockAlarm.alarm.clear({ visibility = { displayed = false }}))
```

Comment:
you probably want to leave the visibility here, since this clear is coming from an actual report from the lock, we'll want it to be visible in history.

Reply:
the "clear" comes from door lock has locked normally.
but to make `lockAlarm` as an event - not as a state -, I asked @seojune79 to hide clear event.
lockAlarm doesn't shown up at detail view and automation. but will send push notification when issued (except `clear` event).
If you strongly suggest `clear` should be shown at history. please let me know

Reply:
So how is `lockAlarm` displayed to the user?

Ultimately, it's your design decision, but I'd just expect users to find it useful to see when an alarm was cleared in history.

Reply:
I'll leave the details on `STSE-2969` ticket